src/components/ButtonPanel.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Button from './Button';
+
+const buttons = [
+  'AC', '+/-', '%', '÷',
+  '7', '8', '9', 'X',
+  '4', '5', '6', '-',
+  '1', '2', '3', '+',
+  '0', '.', '=',
+];
+
+const ButtonPanel = () => (
+  buttons.map(element => <Button key={element.id} buttonName={element.toString()} />)
+);
```

Comment:
So this becomes something like:
```suggestion
const ButtonPanel = () => {
  return (
    <div className="btnPanel">
      buttons.map(element =>
        <Button key={element.id} buttonName={element.toString()} />)
    </div>
  )
};
```